src/v2/view-builder/internals/BaseForm.js

```diff
@@ -12,8 +13,9 @@ export default Form.extend({
     return loc('oform.title.authenticate', 'login');
   },
   save: loc('oform.next', 'login'),
+  customSavingState: {start : 'formRequestStart', stop: 'formRequestCompleted' },
```

Comment:
are we able to leverage default saving state event?

Reply:
you mean default state event i.e `request` ? Using custom saving state gives us better handling over when all the inputs on siw form are getting disabled and re-enabled.. now we can listen to this events anywhere for this purpose. Not sure if this answer your question.

Reply:
Looks like what you have done is pretty much same with default `hasSavingState` behavior.
https://github.com/okta/okta-ui/blob/master/packages/courage/src/views/forms/BaseForm.js#L262-L275
That's why I'm curious what are benefits

Reply:
yeah there no benefit to that. Just having custom state is more verbose and clear otherwise devs can still use `request` `error` and `sync` default events. I can remove them, if you think it doesn't provide any value.

Reply:
Yeah, let's remove it until default behavior isn't sufficient. My understanding is `hasSavingState` shall be transparent to Application developers at most of time.

Reply:
Done.